docs/build_arm.md

```diff
@@ -1,43 +1,93 @@
-# Building Larq Compute Engine for ARM-based boards
+# Building Larq Compute Engine for ARM-based systems
 This page descibes how to build a Larq Compute Engine (LCE) inference binary
-for 32-bit, as well as 64-bit ARM-based boards.
-[Bazel](https://bazel.build/) is the primary build
-system for LCE. However, since in some occasions Make build system is a more
-convenient build solution, we also provide scripts to build LCE with Make.
+for 32-bit, as well as 64-bit ARM-based systems. You can either build LCE
+natively on your ARM device or cross-compile it from a non-ARM machine.
+[Bazel](https://bazel.build/) is the primary build system for LCE, and it can
+be used to cross-compile a binary for ARM-based systems using a non-ARM host.
+To natively build on an ARM system itself we provide a script that uses the
+Makefile build system, because Bazel it is currently not natively available on
+ARM.
+
+This leaves us with three ways to build LCE, which we recommend in the
+following order:
+1. To cross-compile LCE from a host machine, see the section
+   [Cross-compiling LCE with Bazel](#cross-compiling-lce-with-bazel)
+2. To natively compile LCE, see the section
+   [Building LCE with Make](#building-lce-with-make).
+3. To cross-compile LCE using the Make system for users that do not wish to
+   install Bazel, see the section
+   [Cross-compiling LCE with Make](#cross-compiling-lce-with-make).
 
 In this guide, we use the [LCE benchmark tool](../larq_compute_engine/tflite/benchmark)
-source to build an inference binary.
-See [here](./inference.md) to find out how you can create your own
-custom LCE inference application.
+source to build an inference binary. See [here](./inference.md) to find out how
+you can create your own custom LCE inference application.
+
+Although the Raspberry Pi 3 and Raspberry Pi 4 have 64-bit CPUs, note that the
+popular distribution Raspbian for the Raspberry Pi is a 32-bit OS. In order to
+use the optimized 64-bit kernels of LCE on a Raspberry Pi, a 64-bit OS such as
+[Manjaro](https://manjaro.org/download/#raspberry-pi-4-xfce) should be used.
+
+## Cross-compiling LCE with Bazel
+
+First configure Bazel using the instructions [here](build.md). Make sure you
+have run the `./configure.sh` script as instructed.
 
-## Cross-compiling LCE with Bazel ##
-To cross-compile an LCE inference program for the ARM64 architecture,
-the bazel target needs to be built with the ```--config=aarch64``` flag.
-For example, to build the LCE benchmark tool,
-run the following command from the LCE root directory:
+To cross-compile an LCE inference binary for ARM architectures, the bazel
+target needs to be built with the `--config=rpi` (32-bit ARM) or
+`--config=aarch64` (64-bit ARM) flag. For example, to build the LCE benchmark
+tool, run the following command from the LCE root directory:
 
 ```bash
-bazel build -c opt \
```

Comment:
`-c opt`is the flag recommended by original TF benchmark build guide

Reply:
Yes, but we already add it in our `configure.sh` script by default.